interfaces/css-images.idl

```diff
@@ -0,0 +1,8 @@
+// GENERATED CONTENT - DO NOT EDIT
+// Content of this file was automatically extracted from the
+// "CSS Images Module Level 4" spec.
+// See: https://drafts.csswg.org/css-images-4/
+
+partial namespace CSS {
```

Comment:
Namespaces now supported, but this will have to change after or together with an update of cssom.idl

Reply:
The test actually currently passes.. so it's clearly not asserting that it's a namespace. Will send you a PR.

Reply:
Filed after I reviewed this: https://github.com/web-platform-tests/wpt/issues/12123